pipelines/build-release.yml

```diff
@@ -12,24 +12,28 @@ groups:
 resource_types:
   - name: pull-request
     type: docker-image
+    check_every: 24h
     source:
       repository: jtarchie/pr
 
   - name: s3-iam
     type: docker-image
+    check_every: 24h
     source:
       repository: governmentpaas/s3-resource
       tag: fda60bf4c5f85e96c16f704e128e5ead9e84d30d
 
   - name: semver-iam
     type: docker-image
+    check_every: 24h
     source:
       repository: governmentpaas/semver-resource
       tag: ecbdd201e122b44de99a40ac9f24407c1a43b9a2
 
 resources:
   - name: bosh-release-pr
     type: pull-request
+    check_every: 10m
```

Comment:
I think we want to keep the resources that trigger the pipelines at a 1m interval, otherwise we'll be waiting up to 10 mins after opening or merging a PR for a build to start.

Reply:
That is the debate. The thing is, this is so uncommon to be used, that do we really mind? even if you mind, you can always trigger manually.

Reply:
I think I could leave those as 1m and lets try with the other to 24h. Maybe that is enough.

Reply:
Yeah, let's try with these at 1m for now. We can always revisit if that's not enough.

Reply:
done